sdmetrics/reports/multi_table/_properties/cardinality.py

```diff
@@ -65,37 +67,8 @@ def _generate_details(self, real_data, synthetic_data, metadata, progress_bar=No
             'Error': error_messages,
         })
 
-    def _get_details_for_table_name(self, table_name):
-        """Return the details for the given table name.
-
-        Args:
-            table_name (str):
-                Table name to get the details for.
-
-        Returns:
-            pandas.DataFrame:
-                The details for the given table name.
-        """
-        is_child = self.details['Child Table'] == table_name
-        is_parent = self.details['Parent Table'] == table_name
-        return self.details[is_child | is_parent].copy()
-
-    def get_details(self, table_name=None):
-        """Return the details for the property.
-
-        Args:
-            table_name (str):
-                Table name to get the details for.
-                Defaults to ``None``.
-
-        Returns:
-            pandas.DataFrame:
-                The details for the property.
-        """
-        if table_name is None:
-            return self.details.copy()
-
-        return self._get_details_for_table_name(table_name)
+        if self.details['Error'].isna().all():
+            self.details = self.details.drop('Error', axis=1)
```

Comment:
Is there a way we can move this to the base class? I think we did it in the single table case, since every property will run this.

Reply:
Good point, it's actually done for multi-table here:
https://github.com/sdv-dev/SDMetrics/blob/5f7488a35fd1ebd71e12d8fcd153e54815eb5839/sdmetrics/reports/multi_table/_properties/base.py#L114
So I just removed the lines in [9718600](https://github.com/sdv-dev/SDMetrics/pull/488/commits/97186002d26008a2eb06d08e85d6741623bf8043)